.ci/Jenkinsfile

```diff
@@ -103,3 +196,63 @@ pipeline {
     }
   }
 }
+
+def notifyStatus(def args = [:]) {
+  def releaseManagerFile = args.get('file', '')
+  def analyse = args.get('analyse', false)
+  def subject = args.get('subject', '')
+  def body = args.get('body', '')
+  releaseManagerNotification(file: releaseManagerFile,
+                             analyse: analyse,
+                             slackChannel: "${env.SLACK_CHANNEL}",
+                             slackColor: 'danger',
+                             slackCredentialsId: 'jenkins-slack-integration-token',
+                             to: "${env.NOTIFY_TO}",
+                             subject: subject,
+                             body: "Build: (<${env.RUN_DISPLAY_URL}|here>).\n ${body}")
+}
+
+def publishArtifacts(def args = [:]) {
+  // Copy those files to another location with the sha commit to test them afterward.
+  googleStorageUpload(bucket: getBucketLocation(args.type),
+    credentialsId: "${JOB_GCS_CREDENTIALS}",
+    pathPrefix: "${BASE_DIR}/build/binaries/",
+    pattern: "${BASE_DIR}/build/binaries/**/*",
+    sharedPublicly: true,
+    showInline: true)
+}
+
+def runReleaseManager(def args = [:]) {
+  deleteDir()
+  unstash 'source'
+  googleStorageDownload(bucketUri: "${getBucketLocation(args.type)}/*",
+                        credentialsId: "${JOB_GCS_CREDENTIALS}",
+                        localDirectory: "${BASE_DIR}/build/binaries",
+                        pathPrefix: getBucketPathPrefix(args.type))
+  dir("${BASE_DIR}") {
```

Comment:
```suggestion
  dir("${BASE_DIR}") {
    def mageGoal = args.type.equals('staging') ? 'release-manager-dependencies-release' : 'release-manager-dependencies-snapshot'
    withMageEnv() {
      sh(label: 'create dependencies file', script: "mage ${mageGoal}")
    }
```

Reply:
@michalpristas , this suggestion failed:


```
[2022-11-02T21:19:11.528Z] bash: dev-tools/dependencies: No such file or directory
```

see https://fleet-ci.elastic.co/blue/organizations/jenkins/elastic-agent-shipper-mbp/detail/PR-100/16/pipeline/586#step-630-log-10

any idea?

Reply:
https://github.com/elastic/elastic-agent-shipper/pull/100/commits/bc8a1649385e26011039e76e3604e1d8c3d09155 should help